src/model/configuration_example/training/v1.yaml

```diff
@@ -0,0 +1,14 @@
+accelerator: cuda       # cuda | cpu
+strategy: auto          # dp | ddp | ddp_spawn | xla | deepspeed | fsdp
+stage: 2
+offload_optimizer: false
+offload_parameters: false
+devices: 1
```

Comment:
What is this parameter? If it is number of GPU please use `num_gpus`.

Reply:
done